src/form/Toggle/Toggle.module.css

```
@@ -15,7 +10,20 @@
 
   &.disabled {
     cursor: not-allowed;
-    opacity: 0.8;
+    opacity: var(--toggle-disabled-opacity);
```

Let's add a default here.

```suggestion
    opacity: var(--toggle-disabled-opacity, 0.8);
```